upgrade_tests/test_existance_relations/cli/test_contenthosts.py

```diff
@@ -29,7 +29,7 @@
 
 
 # Tests
-@dont_run_to_upgrade('6.3')
+@dont_run_to_upgrade(['6.3', '6.4'])
```

Comment:
same here...no need of change for 6.4 in 6.2.z

Reply:
@ntkathole , The current implementation in existence Tests mandates for keeping the data for all versions regardless of branch/sat_version.

The plan is to keep only TO_VERSION and FROM_VERSION variants in TO_VERSION branch.

So when I will raise that PR, I will be able to easily cherry-pick in all the branches as the implementation in all branches will be same(it is just to avoid/lessen the conflicts in cherry-pick basically).